apps/sms/style/sms.css

```diff
@@ -1038,6 +1038,21 @@ html[dir="rtl"] .contact-header aside.pack-end {
   float: left;
 }
 
+/* ellipsis direction hack before bug 883884 fixed */
+.suggestion > .name > bdi,
+.contact-header > .name > bdi {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: inline-block;
+  max-width: 100%;
+}
+
+/* margin bottom adjustment for inline-block bdi element */
+.suggestion > .name,
+.contact-header > .name {
```

Comment:
I see that some arabic names are a bit cut in contact prompt with this patch, while there're fine in master - see [3] (it's when you tap on contact in participants panel). Also this name is cut in [1], [2] and [4], but I see the same on master. So we should fix at least [3], but if you can deal with [1], [2] and [4] as well, that would be awesome! :)

[1] http://people.mozilla.org/~azasypkin/rtl-issues/issue-1.png
[2] http://people.mozilla.org/~azasypkin/rtl-issues/issue-2.png
[3] http://people.mozilla.org/~azasypkin/rtl-issues/issue-3.png
[4] http://people.mozilla.org/~azasypkin/rtl-issues/issue-4.png